hw/ip/sram_ctrl/data/sram_ctrl.hjson

```diff
@@ -206,6 +206,7 @@
             resval: 0,
           }
           { bits: "3"
+            hwaccess: "hwo",
```

Comment:
@msfschaffner it looks like this wasn't being read. But let me know if it had another intended purpose.

Reply:
Yeah that is correct, this bit does not have to be exposed to HW.